### DISCUSSION TITLE
Detect which type of a click is present when a player makes a shift click

### DISCUSSION THREAD

**Discussion by theEvilReaper** [Minestom/Minestom#1876] [Help]
Is there a way to detect in an inventory if a click is a left shift click or a right shift click? I know that the ClickType enum contains an entry for shift click but it doesn't tell which one is present. Is it not possible or not yet implemented?

**Comment by TheMeinerLP** [comment]
We made our own fork to solve this.
https://github.com/OneLiteFeatherNET/Microtus/pull/2

Also we decided to do this because the community is split into doing nonsense forks(minestom-ce logger/terminal and extension removement)
And the current PR feedback cycle and maintenance of the main project isn't too good to rely on for the future.

Our main goals are solving issues and including open prs from here. 
We are open to discuss useful issues/PR and everything else to help us as a community to keep this project alive
